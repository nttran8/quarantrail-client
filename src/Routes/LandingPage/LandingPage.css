.landingPage {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding-top: 20px;
  padding-left: 20px;
  padding-right: 20px;
  background-color: rgb(153, 140, 148);
  height: 90vh;
}

.landingPage img {
  width: 180px;
  margin-bottom: 15px;
}

.landingPage .intro {
  color: rgb(255, 255, 255);
}

.landingPage .intro h1 {
  font-size: 16px;
  text-align: center;
  margin-bottom: 5px;
}

.landingPage .intro p {
  line-height: 20px;
  font-size: 14px;
  text-align: center;
}

.introButton {
  margin-top: 20px;
  background-color: rgb(236, 228, 160);
  font-size: 20px;
  width: 90px;
  cursor: pointer;
}

.introButton:hover {
  background-color: rgb(209, 192, 41);
  color: rgb(255, 255, 255);
}

.landingPage .player {
  margin-top: 50px;
  font-size: 18px;
}

@media screen and (min-width: 720px) {
  .landingPage {
    padding-top: 0;
    padding-left: 50px;
    padding-right: 50px;
  }

  .landingPage img {
    width: 350px;
    margin-bottom: 20px;
  }

  .landingPage .intro h1 {
    font-size: 28px;
    margin-bottom: 15px;
  }

  .landingPage .intro p {
    line-height: 40px;
    font-size: 18px;
  }

  .introButton {
    margin-top: 60px;
    font-size: 26px;
    width: 130px;
    padding: 5px;
  }
}

@media screen and (min-width: 1000px) {
  .landingPage {
    padding-left: 120px;
    padding-right: 120px;
  }

  .landingPage img {
    width: 320px;
  }

  .landingPage .intro p {
    line-height: 35px;
  }

  .introButton {
    margin-top: 10px;
  }
}
