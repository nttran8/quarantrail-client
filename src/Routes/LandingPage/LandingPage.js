import React, { Component } from "react";
import PersonContext from "../../Context/PersonContext";
import "./LandingPage.css";

export default class LandingPage extends Component {
  static contextType = PersonContext;
  render() {
    return (
      <div className="landingpage">
        <h1>Corona Trail</h1>
        <p>Covid-19</p>
        <button onClick={this.context.renderUser}>Start</button>
      </div>
    );
  }
}
