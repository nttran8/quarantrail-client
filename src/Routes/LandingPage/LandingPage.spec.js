import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LandingPage from './LandingPage'

describe(`LandingPage Route`, () => {

  it('renders by default', () => {
    const wrapper = shallow(<LandingPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})