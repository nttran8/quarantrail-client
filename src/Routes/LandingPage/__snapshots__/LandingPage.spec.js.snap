// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Route renders by default 1`] = `
<div
  className="landingPage"
>
  <img
    alt="quaran trail - stay home"
    src="logo.svg"
  />
  <div
    className="intro"
  >
    <h1>
      The year was 2020...
    </h1>
    <p>
      ...in a world so similar to ours it was pretty much the same world, a global pandemic swept across the planet and forced everyone to be put in quarantine. It may be easy on day 1 or 2, but a whole week? Month? How long can you last? Find out as you journey through boredom and risk in QuaranTrail.
    </p>
  </div>
  <button
    className="introButton"
  >
    Start
  </button>
</div>
`;
